circle.yml

```diff
@@ -17,6 +17,21 @@ jobs:
         command: |
           codespell --quiet-level=4 --ignore-words=./.codespell-whitelist
 
+  bumpversion:
```

Comment:
Add it as a step in `lint` job because starting the `ethereum/cpp-build-env` container is expensive.

Reply:
Changed.